extensions/wikia/Lightbox/Lightbox.i18n.php

```diff
@@ -73,11 +73,7 @@
 	'lightbox-share-email-label' => 'Email this image to a friend',
 	'lightbox-share-email-page-label' => 'Email this page to a friend',
 	'lightbox-share-email-page-label-address' => 'Email address',
-	'lightbox-share-email-ok-caption' => 'Confirmation',
 	'lightbox-share-email-ok-content' => '{{PLURAL:$1|Email has|Emails have}} been sent.',
```

Comment:
Thanks for the info. What I meant is that changing `userLang` param in this query has no visible effect (but don't trust me for sure, I have near-sightedness condition). :see_no_evil: